terraform/config-params/control-center-post-config/variables.tf

```diff
@@ -69,4 +69,14 @@ variable "netmaker_host_name" {
 
 variable "netmaker_version" {
   description = "netmaker version"
+}
+
+variable "loki_data_expiry" {
+  description = "number of days to expire minio loki bucket data"
```

Comment:
do the bucket expiration settings need to be set here?  or can the application set them themselves?

Reply:
Loki does not handle it , it delegates the purging of old chunks to the storage backend system. 
Lonhorn has provision to delete system backups , but it does not have a provision to run it in a scheduled manner, as per my initial observation.

Reply:
@dfry @sijo5722-2021 is right about the expiry settings. Loki only manages the chunk objects lifecycle on the filesystem (using `compactor` and `table-manager`). The responsibility is delegated to the object store whenever it is being used.